web/flat-web/src/pages/JoinPage/JoinPageMobile.tsx

```diff
@@ -17,6 +18,10 @@ export default function JoinPageMobile({
 }: JoinPageMobileProps): React.ReactElement {
     const { t } = useTranslation();
 
+    const isWeChat = useMemo(
```

Comment:
```suggestion
    const isWeChatBrowser = useMemo(
```

move to components outside